### PR TITLE
optionally disable line break before new footnote

### PR DESCRIPTION
**Pull request by frnsys**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -57,6 +57,8 @@ setting `b:vimfootnotetype`:
 +   `Roman`:  I, II, III...
 +   `star`:   \*, \*\*, \*\*\*...
 
+You can optionally disable line breaks before each footnote by setting `b:vimfootnotelinebreak = 0`.
```
Comment (fmoralesc) [inline README.markdown]: Should be `g:vimfootnotelinebreak`.

Comment (frnsys) [reply README.markdown]: Oops, will update now

**Comment by fmoralesc** [comment]
LGTM. Anything to say, @alerque?

**Comment by alerque** [comment]
Thanks for the contribution @frnsys!

**Pull request merged**